src/dbt_client/dbtFusionCommandIntegration.ts

```
@@ -0,0 +1,496 @@
+          .split("\n")
+          .map((line) => JSON.parse(line.trim()));
+        const previewLine = parsedLines.filter(
+          (line) =>
+            line.hasOwnProperty("data") && line.data.hasOwnProperty("preview"),
+        );
+        if (previewLine.length === 0) {
+          throw new Error("TODO: find SQL error: " + stdout);
+        }
+        const compiledSqlLines = parsedLines.filter(
+          (line) =>
+            line.hasOwnProperty("data") && line.data.hasOwnProperty("sql"),
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)